frontend/types/types.ts

```diff
@@ -382,6 +394,7 @@ export type ReleaseInterface = {
   minor?: boolean
   draft?: boolean
   fileIds: Array<string>
+  comments?: Array<ReviewComment>
```

Comment:
I don't think this should be optional. We can always provide an array, even if it's empty